### ISSUE TITLE
Unsupported backdrop blur on iOS ≤ 17

### ISSUE THREAD

**Issue by asemorales** [IrishMorales/irishmorales.github.io#24]
[Won't fix, logging for documentation]
The CSS backdrop-filter: property is not supported on iOS ≤ 17

Resolution:
Won't fix, since lack of backdrop filter doesn't cause any accessibility issues. On iOS ≤ 17, the navbar will just not show the text underneath, which is acceptable.

Found in #12
